### ISSUE TITLE
New category suggestion: Jumpscare

### ISSUE THREAD

**Issue by DShip91** [ajayyy/SponsorBlock#702]
No one likes getting jumpscared. I'm sure many users would appreciate the ability to block them.

**Comment by sky-lake** [comment]
Something like this could also be useful as a way to flag extremely loud sections of otherwise quiet videos.  (This seems to be a recurring element of what passes for internet "humor" these days.)

**Comment by LIME-S** [comment]
Potentially disturbing scenes could fit into a similar category. so they can be easily skipped, if you dont want to watch them, but are interested in the topic of the video.

**Comment by ccuser44** [comment]
Also for flashy and epileptic shenes would be noice

**Comment by thepwrtank18** [comment]
now that Security Breach is out, there's probably a bit more demand for this

**Comment by rcfox2** [comment]
Commenting to show that there is demand for this

**Comment by QuentinMcCarthy** [comment]
Putting a +1 on this to bump it. I've had countless times in videos I'd like to mark jumpscares or other such things. Jumpscare timestamps tend to get posted in the comments of videos, and some creators even pin those comments, so to be able to flag them in SponsorBlock would help those that don't like jumpscares avoid them without having to comb the comments for timestamps.

**Comment by wolgemoth** [comment]
I would also love this feature

**Comment by ccuser44** [comment]
Bruh :-1:

**Comment by wolgemoth** [comment]
> Out of scope? Addressed by https://wiki.sponsor.ajay.app/w/FAQ/New_categories

The FAQ labels jumpscares as not planned as they "could fall under protecting users' health" ...

I'm not sure that deduction is entirely fair. If jumpscares pose any significant threat to a user's health, it is unlikely they would be allowed on YouTube to begin with. The inclusion of a jumpscare category is just to facilitate the warning or elision of an otherwise distracting or annoying segment, and help improve the viewing experience for people who would prefer to avoid them where possible. It feels to me like that perfectly aligns with Sponsorblock's mission.

Additionally: "SponsorBlock should not be relied upon to save a user from jumpscares as many would remain unmarked."

Isn't that the case for sponsor segments as well? As with all categories, there will not be 100% coverage, but this is ok and people respect the hard work of community labellers.

Perhaps if jumpscares are too specific a category, another more general category that encompasses related disruptions could be proposed? Something like: 

"_Tangential/Startling_"? - Unnecessary tangential segments which may dazzle or startle the user through excessively loud volumes, bright flashes, or cuts to digressive non-diegetic content.

**Comment by rcfox2** [comment]
Also there are already categories marked with a disclaimer stating they're experimental, like filler/tangent

**Comment by relrod** [comment]
Agreed with @wolgemoth. Further:

>  It's also not really that common because creators who include jumpscares generally get a lot of negative feedback.

This is _exactly why_ we want this feature. Because they are rare - by design - and very startling when they happen. Yes, a lot of us would avoid videos that have them, if we knew about them. But we have no way of knowing, which is why something like this extension would be a great solution. It would give a way of knowing about and skipping them and letting people flag them.

@ajayyy Can you please reconsider this? I am sure some of us can work together to build the patch out if you'd be open to merging it.

**Comment by rcfox2** [comment]
I found a plugin on the chrome web store titled Jumpscare Notifier For YouTube which isn't perfect, and since it has an AI image I assume it's probably vibecoded, but:
it absolutely works for most of what we want, the only thing it's missing is a centralized database. 
Instead it has an option to search the comments for ones with lots of timestamps, and you can import everything there.
Then, it'll give you a countdown on-screen until that timestamp is reached, I use a 10 second countdown.
Just spreading awareness, sorry for "advertising" on the github but I think that since this unplanned feature is requested so much we might as well redirect these users to somewhere useful.

**Comment by relrod** [comment]
@ajayyy I'd still like to ask if you'd reconsider this.

In the meanwhile I have been using the sponsor category to submit jumpscares to try to mark/skip them for people when I come across them. But I feel weird about doing that because it mis-categorizes them and as someone who wants to be a helpful submitter, it seems better to have them categorized properly so that people can opt in/out to jump scares (maybe some people don't mind them).

Given that there's already a category system, it feels like adding them should be simple. As I've offered a few times already, I suspect that we in the community would take on writing this if you'd be open to a patch.

**Comment by AcesFullOfKings** [comment]
@relrod 

> In the meanwhile I have been using the sponsor category to submit jumpscares

please do not do this. Those submissions are polluting the database and will get your account banned.

> as someone who wants to be a helpful submitter

Submitting segments in the wrong categories is not helping.

The reasons this category doesn't exist have already been laid out in the wiki link above. It's not a question of a lack of development effort, nor simplicity.

**Comment by relrod** [comment]
> The reasons this category doesn't exist have already been laid out in the wiki link above.

Yes, and I am asking here for the decision to be reconsidered.

**Comment by AcesFullOfKings** [comment]
@relrod do you have examples of videos where a jumpscare should be marked?

**Comment by relrod** [comment]
@AcesFullOfKings I admittedly have not been saving a list of them, I can start doing so going forward as I come across them, if it helps make the case. I will admit that it is rare that I come across them (so building a list might be a slow process), but it is one of the most annoying things to experience.

I cannot find it right now, it might have been taken down, but as an example from some time ago, there was a video that was labeled as the theme song to a popular US childrens TV show, and the uploader of the video thought it would be funny to add a jump scare in the middle of it. That was what originally caused me to check if this extension could block it.

> Those submissions are polluting the database and will get your account banned.

That is not my goal and I'm not trying to pollute the database with garbage. I don't want to make the extension _worse_ for anyone. But I am (genuinely) wondering if you have a better suggestion here (that hopefully isn't "create an extension similar to sponsorblock just for this purpose"). I am aware of extensions such as what @rcfox2 mentioned and something like that can work in the interim. But the semantics that SponsorBlock can offer (centralized database and segments having an end-time) are vastly better than what simply searching the comments for timestamps can achieve.

**Comment by AcesFullOfKings** [comment]
If you've been submitting on them then some example videos would be in your submission history. You can find your history at https://sb.ltn.fi and looking at your userID.

I don't have a better suggestion but I do know that "just submit stuff that doesn't have a category as a sponsor" is *definitely* not the right answer.

**Comment by rcfox2** [comment]
> [@AcesFullOfKings](https://github.com/AcesFullOfKings)  the semantics that SponsorBlock can offer (centralized database and segments having an end-time) are vastly better than what simply searching the comments for timestamps can achieve.

Agreed. It would be nice to have but it seems ajay is not willing to budge.

**Comment by wolgemoth** [comment]
@rcfox2 The alternative would be to create a fork of SponsorBlock that recognises the category. But I'm not sure that splitting the community would be very beneficial... 

A good compromise might be if SponsorBlock allows users to add custom databases to the extension in the settings. This way, the "vanilla" SponsorBlock experience remains exactly as ajay intends, but communities that want to maintain alternative categories have a lightweight, viable way to do that.
